mmv1/products/certificatemanager/TrustConfig.yaml

```diff
@@ -124,3 +126,15 @@ properties:
                   PEM intermediate certificate used for building up paths for validation.
                   Each certificate provided in PEM format may occupy up to 5kB.
                 sensitive: true
+  - !ruby/object:Api::Type::Array
+    name: allowlistedCertificates
```

Comment:
Could you also add a test that updates this field if it is updateable?

Reply:
Done! added the field to the existing update test